keyboards/noxary/268_2/268_2.h

```diff
@@ -25,7 +25,7 @@
  * The second converts the arguments into a two-dimensional array which
  * represents the switch matrix.
  */
-#define LAYOUT( \
+#define LAYOUT_65_ansi_blocker( \
```

Comment:
For the sake of any unmerged user keymaps in the wild, do we want to alias LAYOUT to LAYOUT_65_ansi_blocker for this board?

Reply:
I thought about it, but if its unmerged I don't think we can do anything about it.